target/linux/ramips/image/mt7621.mk

```diff
@@ -1350,6 +1350,32 @@ define Device/zbtlink_zbt-wg3526-32m
 endef
 TARGET_DEVICES += zbtlink_zbt-wg3526-32m
 
+define Device/zbtlink_zbt-wg259-16m
```

Comment:
Please care about sorting. In addition, please create a shared definition, so you don't repeat everything.